### DISCUSSION TITLE
Welcome to EasyCodeIt Discussions!

### DISCUSSION THREAD

**Discussion by TheDcoder** [DcodingTheWeb/EasyCodeIt#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Twombs** [comment]
I'm that guy who is too shy to talk about himself.
I use GitHub for my coding projects and so I can share them.

**Reply by TheDcoder** [reply]
No worries mate, everyone is welcome here!
